src/Catalyst.Core.Modules.Web3/Controllers/Handlers/EthGetBalanceHandler.cs

```diff
@@ -34,7 +33,7 @@ public class EthGetBalanceHandler : EthWeb3RequestHandler<Address, UInt256>
         protected override UInt256 Handle(Address address, IWeb3EthApi api)
         {
             // change to appropriate hash
-            Delta delta = api.DeltaResolver.Latest;
+            var _ = api.DeltaResolver.Latest;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable '_'.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)